### ISSUE TITLE
Crash on getPackageInfo

### ISSUE THREAD

**Issue by mtrezza** [stkent/amplify#188]
### Issue
I get quite frequent crash reports from Crashlytics at:

Environment.java – line 88
com.github.stkent.amplify.Environment.getPackageInfo

```
  try {
       return packageManager.getPackageInfo(packageName, flags); // <-- crash here
   } catch (final PackageManager.NameNotFoundException ignored) {
       return null;
   }
```

Only `PackageManager.NameNotFoundException` is caught, so there must be another exception that is thrown.

Unfortunately Crashlytics does not offer more info other than the file and line number.
I think that is because the crash happens right at app start during initialization of amplify.

Any idea why this happens?

**Comment by mtrezza** [comment]
Seems to be a bug in the OS, so others are also changing to handle any kind of exception, e.g.

https://github.com/getsentry/sentry-java/issues/773

**Comment by stkent** [comment]
Ew, gross (of Huawei). I'll take a look at your PR now!

**Comment by stkent** [comment]
Fixed by #189

**Comment by stkent** [comment]
Hi @mtrezza,

I released your fix for this issue with version 2.2.2 of _amplify_. Please update to this new version and let me know if you have any issues!

Best,

Stuart

**Comment by mtrezza** [comment]
Thanks for the quick merge!
